src/backend/graph.js

```diff
@@ -1,47 +1,47 @@
 // @flow
 
-export type ID = {
-  pluginName: string,
+export type Address = {
   repositoryName: string,
-  name: string,
+  pluginName: string,
```

Comment:
As discussed, we're reordering these because we think that the repository name is the broader concept. Let's also change the serialized form.